pkgs/by-name/go/got/package.nix

```diff
@@ -30,8 +30,6 @@ stdenv.mkDerivation (finalAttrs: {
   buildInputs = [ libressl libbsd libevent libuuid libmd zlib ncurses ]
     ++ lib.optionals stdenv.isDarwin [ libossp_uuid ];
 
-  configureFlags = [ "--enable-gotd" ];
```

Comment:
ℹ️ _"The `--enable-gotd` ./configure flag has been removed; gotd is now built unconitionally [sic]."_ — [got 0.99 changelog](https://gameoftrees.org/releases/portable/CHANGELOG)